README.md

```diff
@@ -320,6 +320,37 @@ The script will also have the following environment variables set:
 * PGA_ERROR_MESSAGE (error message, in the case of the error callback)
 
 
+Helm Chart
+----------
+
+The pganalyze collector uses the OCI ([Open Container Initiative](https://opencontainers.org/)) registry to publish the Helm chart.
+You can install the Helm chart for the collector like the following:
+
+```bash
+helm install pga-collector oci://quay.io/pganalyze/pganalyze-collector --values=myvalues.yml
```

Comment:
@lfittl Ok coming back here to just close the loop about the naming of the chart. With the `helm repo` way, here is how the install process will look like with the current name "pganalyze-collector":

```
helm repo add pganalyze https://charts.pganalyze.com/
helm install pga-collector pganalyze/pganalyze-collector
```

It is repeating pganalyze twice, but I think it's okay, as other providers are repeating the name as well like `helm install newrelic-bundle newrelic/nri-bundle`, `helm install my-datadog-operator datadog/datadog-operator`. With that, I'll keep the name as is.